ec-europa/europa-component-library
# refactor(arial): arial everywhere - INNO-739

# PR description

Use of arial everywhere. Everywhere? yes, everywhere!
All screenshots updated.
Several components didn't have test, so I added them.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:


N/A
